docs/_includes/footer.html

```diff
@@ -25,46 +35,39 @@
-        <div class="footer-text">
-          <p>The AAF Support Tutorials website utilises the code framework from Nectar Research Cloud, a service of the Australian Research Data Commons (ARDC).
-            <a href="https://ardc.edu.au/about/" target="_blank">Read more about the ARDC.</a>
-          </p>
-        </div>
       </div>
-      <div class="col-12 col-sm-3">
-        <h5>External Resources</h5>
+      <div class="col">
+        <h5>Legal</h5>
+        <ul class="list-unstyled footer-text">
+          <li>ABN 13 155 355 685</li>
+          <li>ACN 630 805 319</li>
+          <li>Trade Mark: 169 1608 ®</li>
+          <li><a href="https://trust.aaf.edu.au/wp-content/uploads/AAF_PCY_Privacy-policy_v1.3_20230919-for-web-2.pdf">Privacy Policy</a></li>
+          <li><a href="https://aaf.edu.au/wp-content/uploads/2020/10/AAF_PCY_Privacy-Collection-Notice-2020.pdf">Privacy Collection Notice</a></li>
+        </ul>
+      </div>
+      <div class="col">
+        <h5>Resources</h5>
         <ul class="list-unstyled footer-text">
           <li><a href="https://aaf.edu.au/">AAF Home</a></li>
           <li><a href="https://manager.aaf.edu.au/">Federation Manager</a></li>
           <li><a href="https://validator.aaf.edu.au/">Validator Service</a></li>
           <li><a href="https://status.aaf.edu.au/">Federation Status</a></li>
         </ul>
       </div>
-      <div class="col-12 col-sm-3">
+      <div class="col">
         <h5>Enabled by NCRIS</h5>
-        <div class="col-5 float-right">
+        <div class="col float-right">
           <a href="https://www.education.gov.au/ncris" target="_blank">
             <img class="ncris" src="{{ site.baseurl }}/assets/images/ncris_mono_rev.svg" alt="National Collaborative Research Infrastructure Strategy (NCRIS)">
           </a>
         </div>
-      </div>
-      <div class="col-12 col-sm-3">
-        <h5>Contact Us</h5>
-        <ul class="list-unstyled footer-text">
-          <li><a href="mailto:enquiries@aaf.edu.au">enquiries@aaf.edu.au</a></li>
-          <br>
-          <li><a href="mailto:support@aaf.edu.au">support@aaf.edu.au</a></li>
-          <br>
-          <li>Phone: +61 7 3854 2353</li>
-          <br>
-          <li>ABN 13 155 355 685</li>
-          <li>ACN 630 805 319</li>
-          <li>Trademark: 169 1608 ®</li>
-          <br>
-          <li><a href="https://trust.aaf.edu.au/wp-content/uploads/AAF_PCY_Privacy-policy_v1.3_20230919-for-web-2.pdf">Privacy Policy</a></li>
-          <li><a href="https://aaf.edu.au/wp-content/uploads/2020/10/AAF_PCY_Privacy-Collection-Notice-2020.pdf">Privacy Collection Notice</a></li>
-        </ul>
+        <br>
+        <div class="footer-text set-width">
+          <p>The AAF Support Tutorials website utilises the code framework from Nectar Research Cloud, a service of the Australian Research Data Commons (ARDC).
+            <a href="https://ardc.edu.au/about/" target="_blank">Read more about the ARDC.</a>
```

Comment:
This spacing looks a little odd on the screenshots, maybe can just drop the `set-width` class as the other message aroundm the traditional custodians looks fine?

```suggestion
        <div class="footer-text">
          <p>The AAF Support Tutorials website utilises the code framework from Nectar Research Cloud, a service of the Australian Research Data Commons (ARDC).
            <a href="https://ardc.edu.au/about/" target="_blank">Read more about the ARDC.</a>
```

Reply:
this is what it looks like with the set-width class:
<img width="665" alt="Screenshot 2025-01-22 at 3 28 28 pm" src="https://github.com/user-attachments/assets/9aed6327-becb-4f01-8110-4947d6e61311" />

this is what it looks like without:
<img width="665" alt="Screenshot 2025-01-22 at 3 29 11 pm" src="https://github.com/user-attachments/assets/da1912cc-7d58-48b4-a123-3101e96f4b93" />

Reply:
ah I see, all good